src/commands/info.js

```diff
@@ -1,16 +1,45 @@
 // @flow
 import * as options from '../utils/options';
 import { BoltError } from '../utils/errors';
+import * as yarn from '../utils/yarn';
 
-export type InfoOptions = {};
+export type InfoOptions = {|
+  cwd?: string,
+  args: options.Args
+|};
+
+function flagsToArgs(flagArgs: options.Flags): Array<string> {
+  const flagArgsKey = Object.keys(flagArgs);
+  // Flags has default -- key
+  if (flagArgsKey.length < 1) {
```

Comment:
It should be `<= 1`  🙂

Reply:
@ajaymathur I removed this piece of code.